applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp

```diff
@@ -72,6 +72,9 @@ bool SmallStrainIsotropicDamage3D::Has(const Variable<Vector>& rThisVariable)
 {
     if(rThisVariable == INTERNAL_VARIABLES){
         return true;
+    } else if(rThisVariable == STRAIN){
+        // explicitly returning "false", so the element calls CalculateValue(...)
+        return false;
```

Comment:
@RiccardoRossi : I explicitly set it to "false" to indicate that it is actually implemented (in CalculateValue), and to avoid the confusion of setting it to true (for GetValue) when is meant to be in CalculateValue, which has given problems in the past.